_partners/calyptia.md

```diff
@@ -0,0 +1,6 @@
+---
+name: Calyptia
+logo: '/assets/media/partners/calyptia.png'
```

Comment:
Would you be able to pad out the logo to make it square (or do you have a square/rectangle version)?

If not, would you mind if we pad it out for you?
![Screen Shot 2022-01-19 at 3 42 07 PM](https://user-images.githubusercontent.com/1152927/150230152-98fb9cfe-c8e9-42d9-9ddd-72b384808603.png)